content/knitr/options.md

```diff
@@ -6,79 +6,79 @@ slug: options
 show_toc: true
 ---
 
-The **knitr** package shares most options with Sweave, but some were
-dropped or changed, and some new options were added. The default values are in
-the parentheses below. Note that the chunk label for each chunk is assumed
-to be unique, i.e., no two chunks should share the same label. This is especially
+The **knitr** package has most of the same options as Sweave, but some were
+dropped or changed and some new options were added. The default values are in
+parentheses below. Note that the chunk label for each chunk is assumed
+to be unique, i.e. no two chunks should share the same label. This is especially
```

Comment:
It is no longer necessary to mention Sweave here: knitr was created to replace Sweave, and I guess it has done its job long time ago :)